app/modules/logoutNotifier/logoutNotifierController.js

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright (c) 2016 eyeOS
+
+ This file is part of Open365.
+
+ Open365 is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+"use strict";
+
+define([], function () {
+	function LogoutNotifierController(logoutNotifierService, $translate, bootstrapDialog) {
+		this.logoutNotifierService = logoutNotifierService;
+		this.$translate = $translate;
+		this.bootstrapDialog = bootstrapDialog || window.BootstrapDialog;
+	}
+
+	LogoutNotifierController.prototype.init = function () {
+		this.logoutNotifierService.addLogoutCallback(this.notifyLogout.bind(this));
+	};
+
+	LogoutNotifierController.prototype.notifyLogout = function () {
+		$('body').addClass('modal_mode_on');
```

Comment:
Could we have this "addClass" and "show" function grouped in a same method?
I mean, create a eyeosDialog object with a showModal method which:

1. Adds the class modal
2. calls bootstrapDialog.show